modin/pandas/test/utils.py

```diff
@@ -162,6 +162,15 @@
     }
 }
 
+test_data_middle = {
+    "int_data": {
+        "col{}".format(int((i - 18) % 37)): random_state.randint(
+            RAND_LOW, RAND_HIGH, size=(34)
+        )
+        for i in range(36)
+    },
+}
+
```

Comment:
why do we need to generate new test set?

Reply:
We need to generate new small test set. Reason: result of `unstack` for use-case described in #1997 will generate dataframe-result of size `NROWS^nlevels*(len(columns.level(0)*..*len(columns.level(i))`. Testing of data from `test_data` takes about 10 minutes for one `df_equals` (all time is spent on `df_equals`).

Reply:
then we can just cut already existing data from `test_data` inside test, for example
```python
def test_unstack():
    MAX_NROWS = 36
    MAX_NCOLS = 34

    md_df = pd.DataFrame(test_data["int_data"]).iloc[:MAX_NROWS, :MAX_NCOLS]
```

Reply:
Thanks! It was applied.